Tests/SwiftGRPCTests/ChannelArgumentTests.swift

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2018, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#if SWIFT_PACKAGE
+import CgRPC
+#endif
+import Foundation
+@testable import SwiftGRPC
+import XCTest
+
+class ChannelArgumentTests: XCTestCase {
+  func testArgumentKey() {
+    let argument: Channel.Argument = .defaultAuthority("default")
+    XCTAssertEqual(String(cString: argument.toCArg().key), "grpc.default_authority")
```

Comment:
Pretty sure that these (and below) leak memory, though, but OTOH they are only one-of tests. @timburks, what do you think?